onfido-java/src/test/java/com/onfido/integration/CheckManagerTest.java

```diff
@@ -1,222 +1,141 @@
 package com.onfido.integration;
 
 import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
 
 import com.onfido.JsonObject;
 import com.onfido.Onfido;
 import com.onfido.api.FileDownload;
+import com.onfido.models.Applicant;
 import com.onfido.models.Check;
+import com.onfido.models.Document;
+
 import java.util.Arrays;
 import java.util.List;
 
 import com.onfido.models.DrivingLicence;
+
 import okhttp3.mockwebserver.MockWebServer;
 import okhttp3.mockwebserver.RecordedRequest;
+import org.testng.annotations.BeforeMethod;
 import org.testng.annotations.Test;
+
 import java.util.ArrayList;
 
-public class CheckManagerTest extends ApiIntegrationTest {
+public class CheckManagerTest extends TestsHelper {
+  private Applicant applicant;
+  private Document document;
 
-  @Test
-  public void createCheck() throws Exception {
-    String response =
-        new JsonObject()
-            .add("applicant_id", "id")
-            .add("webhook_ids", null)
-            .add("privacy_notices_read_consent_given", true)
-            .toJson();
-
-    MockWebServer server = mockRequestResponse(response);
-
-    Onfido onfido =
-        Onfido.builder().apiToken("token").unknownApiUrl(server.url("/").toString()).build();
-
-    Check check =
-        onfido.check.create(Check.request().applicantId("id").reportNames("report_name_1").privacyNoticesReadConsentGiven(true));
-
-    // Correct path
-    RecordedRequest request = server.takeRequest();
-    assertEquals("/checks/", request.getPath());
-
-    // Correct request body
-    String json = request.getBody().readUtf8();
-    JsonObject jsonObject = JsonObject.parse(json);
-    assertEquals("id", jsonObject.get("applicant_id"));
-    assertEquals(true, jsonObject.get("privacy_notices_read_consent_given"));
-
-    // Correct response body
-    assertEquals("id", check.getApplicantId());
-    assertEquals(null, check.getWebhookIds());
-    assertEquals(true, check.getPrivacyNoticesReadConsentGiven());
+  @BeforeMethod
+  public void setup() throws Exception {
+    applicant = createApplicant();
+    document = uploadDocument(applicant, "file.png");
```

Comment:
We sure it is `file.png` ❓ 😃

Reply:
That's just the fine name provided to the API but it doesn't need to be the actual filename we're uploading.
It's provided as a parameter to be able to distinguish among different file in `DocumentManagerTest::listDocumentsTest()` function.